Cargo.toml

```diff
@@ -168,28 +168,28 @@ revm-inspectors = { git = "https://github.com/paradigmxyz/revm-inspectors", rev
 
 ## alloy
-alloy-consensus = { version = "0.1.1", default-features = false }
-alloy-contract = { version = "0.1.1", default-features = false }
-alloy-eips = { version = "0.1.1", default-features = false }
-alloy-genesis = { version = "0.1.1", default-features = false }
-alloy-json-rpc = { version = "0.1.1", default-features = false }
-alloy-network = { version = "0.1.1", default-features = false }
-alloy-node-bindings = { version = "0.1.1", default-features = false }
-alloy-provider = { version = "0.1.1", default-features = false }
-alloy-pubsub = { version = "0.1.1", default-features = false }
-alloy-rpc-client = { version = "0.1.1", default-features = false }
-alloy-rpc-types = { version = "0.1.1", default-features = false }
-alloy-serde = { version = "0.1.1", default-features = false }
-alloy-signer = { version = "0.1.1", default-features = false }
-alloy-signer-local = { version = "0.1.1", default-features = false }
-alloy-signer-aws = { version = "0.1.1", default-features = false }
-alloy-signer-gcp = { version = "0.1.1", default-features = false }
-alloy-signer-ledger = { version = "0.1.1", default-features = false }
-alloy-signer-trezor = { version = "0.1.1", default-features = false }
-alloy-transport = { version = "0.1.1", default-features = false }
-alloy-transport-http = { version = "0.1.1", default-features = false }
-alloy-transport-ipc = { version = "0.1.1", default-features = false }
-alloy-transport-ws = { version = "0.1.1", default-features = false }
+alloy-consensus = { version = "0.1", default-features = false }
+alloy-contract = { version = "0.1", default-features = false }
+alloy-eips = { version = "0.1", default-features = false }
+alloy-genesis = { version = "0.1", default-features = false }
+alloy-json-rpc = { version = "0.1", default-features = false }
+alloy-network = { version = "0.1", default-features = false }
+alloy-node-bindings = { version = "0.1", default-features = false }
+alloy-provider = { version = "0.1", default-features = false }
+alloy-pubsub = { version = "0.1", default-features = false }
+alloy-rpc-client = { version = "0.1", default-features = false }
+alloy-rpc-types = { version = "0.1", default-features = false }
+alloy-serde = { version = "0.1", default-features = false }
+alloy-signer = { version = "0.1", default-features = false }
+alloy-signer-local = { version = "0.1", default-features = false }
+alloy-signer-aws = { version = "0.1", default-features = false }
+alloy-signer-gcp = { version = "0.1", default-features = false }
+alloy-signer-ledger = { version = "0.1", default-features = false }
+alloy-signer-trezor = { version = "0.1", default-features = false }
+alloy-transport = { version = "0.1", default-features = false }
+alloy-transport-http = { version = "0.1", default-features = false }
+alloy-transport-ipc = { version = "0.1", default-features = false }
+alloy-transport-ws = { version = "0.1", default-features = false }
```

Comment:
fully specifying 0.1.X here makes sense because we might have breaking changes or we depend on new features, pls revert this and set to 0.1.2

Reply:
Agreed, reverted and pinning to 0.1.2